docs/docs/data/datasets.md

```diff
@@ -37,17 +37,16 @@ for stability**.
 
 ### 1. **Fork the OpenAssistant repository**
 
-Fork the
-`OpenAssistant`[repository](https://github.com/LAION-AI/Open-Assistant). To do
-this, click the link to the repository and click "Fork" in the upper-right
+Fork the [OpenAssistant repository](https://github.com/LAION-AI/Open-Assistant).
+To do this, click the link to the repository and click "Fork" in the upper-right
 corner. You should get an option to fork to your account, provided you are
 signed into Github.
 
 After you fork, clone the repository locally. You can do so as follows:
 
 ```bash
 git clone git@github.com:<your_github_username>/OpenAssistant.git
```

Comment:
we should probably also update this line?

Reply:
Done